app/src/components/Category.js

```diff
@@ -0,0 +1,30 @@
+import styled, { ThemeProvider } from "styled-components";
+import theme from "../styles/theme.js";
+import CardList from "./CardList.js";
+
+const Header = styled.h3`
+  font-size: ${({ theme }) => theme.fontSize.medium};
+`;
+
+const Category = ({ dish, category }) => {
+  const title = {
+    side: "식탁을 풍성하게 하는 정갈한 밑반찬",
+    soup: "정성이 담긴 뜨끈뜨끈 국물 요리",
+    main: "모두가 좋아하는 든든한 메인 요리",
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Header>{title[category]}</Header>
+      <CardList
+        key={category}
+        cards={dish}
+        tabNumber={1}
+        index={1}
+        cardSize={"medium"}
+      ></CardList>
```

Comment:
태그 사이에 써줄 게 없다면 줄여서 표현할 수 있습니다.
또한 스트링을 넘겨줄 때에는 중괄호를 생략 가능합니다.
```suggestion
      <CardList
        key={category}
        cards={dish}
        tabNumber={1}
        index={1}
        cardSize="medium" />
```